setup.py

```diff
@@ -458,6 +455,11 @@ def pxd(name):
            'sources': ['pandas/src/datetime/np_datetime.c',
                        'pandas/src/datetime/np_datetime_strings.c',
                        'pandas/src/period.c']},
+    period=dict(pyxfile='period',
```

Comment:
I guess I liked it better this way, and ext_data['parser'] is also like that. Do you want me to change that one too?